jhove-apps/src/main/java/edu/harvard/hul/ois/jhove/viewer/RepTreeRoot.java

```diff
@@ -71,6 +71,26 @@ public RepTreeRoot(RepInfo info, JhoveBase base) {
 		snarfRepInfo();
 	}
 
+	private static final String TEXT_MD_METADTA = "TextMDMetadata";
+	private static final String FORMAT = "Format: ";
+	private static final String VERSION = "Version: ";
+	private static final String BYTE_ORDER = "ByteOrder: "
```

Comment:
This line's lacking a trailing semicolon that breaks the build: https://travis-ci.org/github/openpreserve/jhove/jobs/684731746#L4367

Reply:
Added! Sorry about that!